src/TriggerBinding/SqlTriggerMetrics.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace Microsoft.Azure.WebJobs.Extensions.Sql
 {
-    internal sealed class SqlTriggerMetrics : ScaleMetrics
+    public class SqlTriggerMetrics : ScaleMetrics
```

Comment:
Because SqlTriggerScaleMonitor does not need to be public, change back to internal sealed